rosdep/python.yaml

```diff
@@ -5465,6 +5465,9 @@ python-ujson:
     vivid:
       pip:
         packages: [ujson]
+python3-ujson:
+  ubuntu:
+    '*': [python3-ujson]
```

Comment:
@jknofe I see `python3-ujson` available in other [supported platforms](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules) as well. Would you mind adding the a few more entries?

Reply:
@hidmic thanks for your review, I checked most of the other distributions:

**Debian**: I propose to use the '*' wildcard also. 
https://packages.debian.org/sid/python3-ujson

**Fedora**, pkg is available and added
https://packages.fedoraproject.org/pkgs/python-ujson/python3-ujson/


My second commit is for Gentoo, Nixos and OSX. I added ujson same way as for python-ujson. This might help to keep a pkg using python3-ujson portable, but these 3 are not going to distinguish between python3 and python2 version of the same pkg.